.travis.yml

```diff
@@ -63,10 +64,27 @@ jobs:
       install: npm run bootstrap
       after_success: npm run coveralls
       env: 'COVERAGE=true'
-
-    # - &node
-    #   script: npm run test
-    #   node_js: 'lts/*'
+    
+    - stage: deploy
+      before_install:
+      - mkdir $HOME/.aragon
+      - openssl aes-256-cbc -K $encrypted_547551ebf6e1_key -iv $encrypted_547551ebf6e1_iv
+        -in shared/deployments/rinkeby_key.json.enc -out $HOME/.aragon/rinkeby_key.json -d
+      - ls $HOME/.aragon
+      install: 
+        - travis_wait npm i -g @aragon/cli@5.6.2
+        - npm run bootstrap
+      script: echo deploying if running in dev...
+      deploy:
+        provider: script
+        script: travis_wait 60 npm run publish:cd
+        on:
+          branch: dev
 
 notifications:
   email: false
+
+# Only build pushes to dev. All pull requests still build
+branches:
+  only:
+  - dev
```

Comment:
We've been having double-build problems on our PRs lately, with Travis configured to run on _every branch push_ and on _every push to a PR_. This means that pushes to branches associated with PRs build twice. Will this configuration fix that problem?

Reply:
I recently enabled the pushes build again for some builds I wanted to test, related to `mainnet-audit-*` branches, since those were not directly targeting `dev`. I disabled it again, I think it is easy enough to enable-disable it on demand for things like that, do we need to have a static setting for this config to work @topocount ?

Reply:
Yes this will resolve this and enable push-builds to only execute on merges to dev (as the comment on line 87 indicates). We'll be able to turn them on and not have them run concurrently with PR builds.

Also @chadoh even though it is resource-intensive, the double-builds provide some additional value in that the PR build has the parent codebase merged into the child branch. This is why PR builds won't execute if there are merge conflicts. The push-builds though just take the branch as-is and test it though. Executing both of these saves some time in terms of testing and debugging, but if the developer is already testing their code locally, the push-builds are mostly unnecessary